src/main/kotlin/com/vk/kphpstorm/configuration/KphpStormStartupActivity.kt

```diff
@@ -20,7 +23,48 @@ class KphpStormStartupActivity : ProjectActivity {
     }
 
     private fun showSetupDialog(project: Project) {
-        if (!KphpStormConfiguration.wasSetupForProject(project) && KphpStormConfiguration.seemsLikeProjectIsKphpBased(project))
+        if (!KphpStormConfiguration.wasSetupForProject(project) && KphpStormConfiguration.seemsLikeProjectIsKphpBased(project)){
             SetupPluginForProjectDialog(project).show()
+        }else{
+            showNotification(project)
```

Comment:
In this case, the notification will appear even if the project **is configured**

Reply:
Fixed